mpmath/matrices/matrices.py

```diff
@@ -362,28 +363,21 @@ def __nstr__(self, n=None, **kwargs):
     def __str__(self):
         return self.__nstr__()
 
-    def _toliststr(self, avoid_type=False):
```

Comment:
Removed the extra parameter here because this private method is invoked only from `__repr__()`  with the value set to `True